XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
I meant if we leave one unified target (for macOS, iOS) will Swift Package Manage be able compiling it for macOS and iOS platforms? e.g. I think that Carthage won't be able doing it (but I'm not sure).